data/coalition/coalition missions.txt

```diff
@@ -250,7 +250,7 @@ mission "Kimek Student 2"
 					decline
 			label accept
 			`	He thanks you, and this time follows you right away, as he already has his bags prepared.`
-			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he isn't aiming for the more military-focused side of the government.`
+			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work for for the scientific branch of the government, instead of the more military-focused side.`
```

Comment:
```suggestion
			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work for the scientific branch of the government, instead of the more military-focused side.`
```